docs/source/modin/supported/series_supported.rst

```diff
@@ -100,7 +100,8 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``argsort``                 | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``asfreq``                  | N                               |                                  |                                                    |
+| ``asfreq``                  | P                               | ``how``, ``normalize``,          | See ``resample``                                   |
```

Comment:
Can we test if those arguments are used, we raise NotImplementedError?

Reply:
Good point, added a block to raise NotImplementedError for unused arguments. If an aggregation that is not supported is passed in (like `bfill`), the resample code itself will ensure that NotImplementedError is raised.